backend/manager/modules/restapi/types/src/main/java/org/ovirt/engine/api/restapi/types/StorageLogicalUnitMapper.java

```diff
@@ -48,6 +48,11 @@ public static LogicalUnit map(LUNs entity, LogicalUnit template) {
         model.setSize(SizeConverter.convert((long)entity.getDeviceSize(),
                 SizeConverter.SizeUnit.GiB, SizeConverter.SizeUnit.BYTES).longValue());
 
+        if (entity.getLunConnections() != null && !entity.getLunConnections().isEmpty()) {
+            model.setAddress(entity.getLunConnections().get(0).getConnection());
+            model.setPort(Integer.valueOf(entity.getLunConnections().get(0).getPort()));
```

Comment:
why not having the whole details of `StorageServerConnections`? we will need the iqn(target) for example